src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt

```diff
@@ -71,7 +71,7 @@ object Messages {
         "We're no strangers to code\nYou read the rules and so will I\nA git commit is what I'm thinking of\nYou wouldn't get this from %name%",
         "%name% has made the wise decision to hide behind a bush. However, they have also made the common mistake of hiding in the only bush in the area. This is how not to be seen.",
         "May I have your attention, please?\nWill the real %name% please stand up?\nI repeat, will the real %name% please stand up?\nWe're gonna have a problem here",
-        "My name is %name%, Commander of the coders of the North, General of the IDE legions, loyal servant to the true emperor, Open Source. Father to an unfinished project, husband to an idea waiting to be started. And I will have my vengeance, in this life or the next.",
+        "My name is %name%, Commander of the coders of the North, General of the IDE legions, loyal servant to the true emperor, Open Source. Owner to an unfinished project, partner to an idea waiting to be started. And I will have my vengeance, in this life or the next.",
```

Comment:
Grammatically, this should probably be "Owner of an unfinished project".

Reply:
I think "Parent to an unfinished project" would fit more with the original theme of the quote, whilst keeping with the gender neutral tone

Reply:
Yeah, that sounds good.